spec/board_spec.rb

```diff
@@ -77,4 +77,62 @@
       expect(board.full?).to eq(true)
     end
   end
+
+  describe '#vertical_win?' do
+    it 'returns true if column has four consecutive X symbols' do
+      board = Board.new
+      # refactor below later
+      board.layout[:A] = ['X','X','X','X','O','O'],
+      board.layout[:B] = ['X','X','X','X','O','X'],
+      board.layout[:C] = ['X','O','X','X','X','X'],
+      board.layout[:D] = ['O','O','X','X','X','X'],
+      board.layout[:E] = ['X','X','X','X','O','O'],
+      board.layout[:F] = ['O','X','X','X','X','O'],
+      board.layout[:G] = ['X','X','X','X','O','O']
+
```

Comment:
Change request: update board to reflect real game (i.e., no more than one column should have four consecutive pieces)

Reply:
thanks!